src/components/MoneyRequestConfirmationList.js

```diff
@@ -150,9 +150,15 @@ const propTypes = {
     /** Whether the money request is a scan request */
     isScanRequest: PropTypes.bool,
 
+    /** Whether we're editing a split bill */
+    isEditingSplitBill: PropTypes.bool,
+
     /** Whether we should show the amount, date, and merchant fields. */
     shouldShowSmartScanFields: PropTypes.bool,
 
+    /** Whether we should show the amount, date, and merchant fields. */
+    isScrollable: PropTypes.bool,
```

Comment:
I think we can remove this prop and make this component always scrollable.
And instead remove scroll views in `MoneyRequestConfirmPage`

Reply:
I don't think we can do that. It would alter the behavior in the search page where we want the search bar to remain fixed on top.

Reply:
This component is used only in 2 places - SplitBillDetailsPage, MoneyRequestConfirmationPage
There's no search bar

Reply:
Ah yes sorry. I mistakenly thought the comment was under BaseOptionsSelector